app/inbound/calculation/get-calculation-by-sbi.js

```diff
@@ -0,0 +1,13 @@
+const db = require('../../data')
+
+const getCalculationBySbi = async (sbi, transaction) => {
+  return db.organisation.findOne({
+    transaction,
+    lock: true,
```

Comment:
What's the value in locking the rows if we're not going to update the locked record?

Reply:
Again, naively gust copied from other db retrievals, inserts, etc -- will remove as I assume this is only used for data entry

Reply:
Yeah you can lock it if you're going to update that record.  If you're not going to update, no value in the lock.  Because you're running in a transaction _shouldn't_ cause any issues but it adds no value here.